examples/online_coding/config/qwen2.5-1.5b-ppo.yaml

```diff
@@ -0,0 +1,298 @@
+defaults:
+  - override hydra/job_logging: stdout
+
+hydra:
+  run:
+    dir: .
+  output_subdir: null
+
+cluster:
+  num_nodes: 1
+  num_gpus_per_node: 8
+  component_placement:
+    rollout: 0-3
+    inference: 4-5
+    actor: 6-7
+
+runner:
+  task_type: math
+  logger:
+    log_path: ${runner.output_dir}/${runner.experiment_name}
+    project_name: rlinf
+    experiment_name: ${runner.experiment_name}
+    logger_backends: ["tensorboard"] # wandb, swanlab
+
+  max_epochs: 10
+  max_steps: -1
+
+  val_check_interval: 1
+  save_interval: 10
+
+  seq_length: 2560
+
+  enable_dynamic_batch_size: False
+  max_tokens_per_mbs: 2560
+
+  resume_dir: null
+  experiment_name: online-ppo-1.5b-pipeline
+  output_dir: /mnt/public/zhuchunyang_rl/logs
+
+algorithm:
+  group_size: 1
+
+  n_minibatches: 2
+  training_batch_size_per_gpu: 1 # micro batch size
+  rollout_batch_size_per_gpu: null # If set to null, rollout_batch_size will be evenly divided across all inference instances. You can reduce this parameter if inference consumes too much GPU memory.
+
+  # mbs to do log prob inference, can be set to
+  # lower than rollout_batch_size_per_gpu to reduce
+  # memory usage
+  logprob_forward_micro_batch_size: 1 # ${.rollout_batch_size_per_gpu}
+
+  # val rollout mbs
+  val_rollout_batch_size_per_gpu: 4 # ${.rollout_batch_size_per_gpu}
+
+  recompute_logprobs: True
+  shuffle_rollout: False
+
+  max_num_gen_batches: 1
+
+  # PPO loss params (no critic model)
+  loss_type: math_ppo_actor
+  loss_agg_func: "token-mean"
+  kl_beta: 0.0 # 0.001
+  kl_penalty_type: low_var_kl
+  ratio_clip_eps: 0.2
+  entropy_bonus: 0.0
+  calculate_entropy: False
+  clip_ratio_c: null # 3.0
+
+  # Control critic usage (similar to AReaL's disable_head)
+  use_critic: false  # Disable critic model
+  use_value_loss: false  # Disable value loss computation
+  
+  # PPO parameters for no-critic setup
+  gamma: 0.99
+  gae_lambda: 0.95
+  # value_clip and huber_delta not needed without critic
+
+  # Use no-critic GAE advantage computation
+  adv_type: math_gae_no_critic  # Will implement this function
```

Comment:
here is `math_gae`, so this coding rl training uses the same `adv_type` as math rl?